specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx

```diff
@@ -286,6 +286,14 @@ function EditRecordDialog<SCHEMA extends AnyTree>({
   );
 }
 
+const frontendToBackendMappingActions = {
+  bulkMove: 'bulk_move',
+  desynonymize: 'desynonymize',
+  merge: 'merge',
+  move: 'move',
+  synonymize: 'synonymize',
+};
+
```

Comment:
Instead of redefining every `Action` in this Object (and losing type safety), you can likely leverage the spread operator and the action type to make this more accessibile for future developers. 

Consider the following modifications which can be made to the top of the file (and modifying the `Action` type): 

```ts
const treeActions = [
  'add',
  'bulkMove',
  'desynonymize',
  'merge',
  'move',
  'synonymize',
] as const;

type Action = typeof treeActions[number];

const frontendToBackendMappingActions: Readonly<Record<Action, string>> = {
  ...Object.fromEntries(treeActions.map((action) => [action, action])),
  bulkMove: 'bulk_move',
};
```

Now with the `...Object.fromEntries(treeActions.map((action) => [action, action]))`, we only need to specify the actions which need to be explicitly overwritten. Even if other Actions are added, modified, or removed, the mapping object gets updated automatically.

Additionally, this approach allows the following to be simplified to:

https://github.com/specify/specify7/blob/58ab2b2ae7b09ba13433bc9b8bd9d8de8c3a2080/specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx#L289-L290

```ts
  if (!treeActions.includes(type)) throw new Error('Invalid action type');
```